.github/workflows/e2e.yml

```diff
@@ -0,0 +1,57 @@
+name: e2e Tests
+on:
+  push:
+    branches: [main]
+    paths-ignore:
+      - 'docs/**'  # only trigger build if a file outside of /docs was changed
+      - '_locales/*.xml'
+      - '.vscode/**'
+      - 'README_ASSETS/**'
+      - 'README.md'
+  pull_request:
+    paths-ignore:
+      - 'docs/**'  # only trigger build if a file outside of /docs was changed
+      - '_locales/*.xml'
+      - '.vscode/**'
+      - 'README_ASSETS/**'
+      - 'README.md'
+
+
+
+jobs:
+  test:
+    name: E2E Tests
+    environment: test
+    env:
+      CI: true
+      NODE_ENV: test
+      PRIVATE_CERTIFICATE_KEY: ${{ secrets.PRIVATE_CERTIFICATE_KEY }}
+      PRIVATE_CERTIFICATE_CERT: ${{ secrets.PRIVATE_CERTIFICATE_CERT }}
+      VERSION_INFO_GIT_REF: ${{ github.ref }}
+      DC_ACCOUNTS_DIR: ../../e2e-tests/data/accounts
+    timeout-minutes: 60
+    runs-on: ${{ matrix.os }}
```

Comment:
I doubt operating system is relevant for browser. more interesting might be different browsers. with electron it is only chrome, but with tauri it is chrome and webkit/safari

Reply:
we can pass no project parameter to pnpm e2e, then it runs on all configured browsers. right now, (Firefox, Chrome & Chromium)

Reply:
webkit might also work, didn't try yet

Reply:
may be easiest to run safari on macOS runner?

Reply:
webkit does not allow clipboard read in my test. Needs further investigation
might be related to https://github.com/microsoft/playwright/issues/13037